core/schema/host.graphqls

```diff
@@ -6,32 +6,19 @@ extend type Query {
 "Information about the host execution environment"
 type Host {
   "The current working directory on the host"
-  workdir: HostDirectory!
+  workdir: Directory!
 
-  "Access a directory on the host"
-  directory(id: HostDirectoryID!): HostDirectory!
+  "The root filesystem on the host"
+  fs: Directory!
```

Comment:
This feels like a major security risk, and wouldn't it mean uploading their entire hard drive? I'm not sure I'd trust any extensions at this point.

Reply:
Extensions would definitely *not* have access to the real host... Each extension would have its own sandboxed "host" with its own content and nothing else. Either that, or privileged host access would simply return an error when an extension calls it. Otherwise, yeah, 100% agree it would be a major problem. It's very important that extensions can be *safely* shared and reused.